Server/sprawl_calculator/urban_sprawl/clip_raster/raster_operations.py

```diff
@@ -0,0 +1,21 @@
+import rasterio
+from rasterio.mask import mask
+
+
+def clip_raster_with_geojson(raster_file, geojson):
```

Comment:
The type of raster file is a Temporary file, so can we add typing here e.g. `raster_file: NamedTemporaryFIle` and import that higher as well.